__tests__/app.test.js

```diff
@@ -21,3 +22,13 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET: 200 responds with an object describing all available endpoints on API", () => {
+    return request(app)
+      .get("/api")
+      .then(() => {
+        expect(endpoints).toEqual(endpoints);
```

Comment:
this test isn't proving anything really, you're just checking that the json looks like the json. 

you want to check that what comes back on the body of the request looks like the json. give your .then a parameter of result and then have a look at result.body